### ISSUE TITLE
New icons added to repo are not visible immediately

### ISSUE THREAD

**Issue by erikng** [hjuutilainen/munkiadmin#80]
Not sure if this should even be considered a bug per se.

1. Add new icon because MunkiAdmin shows you just how ugly your repo has become
2. Icon is not searchable through pkginfo tool or right click -> icon tool.

Can be fixed by
Hand writing icon name "Foo.png" into pkginfo tool.

Icon is now available and searchable through both methods.

**Comment by erikng** [comment]
Also not sure how to even fix this or if it's really worth fixing.

**Comment by hjuutilainen** [comment]
So you mean you manually copy a icon file to the icons directory? I'd say this is expected behavior. MunkiAdmin currently resolves these at the scanning phase and doesn't watch the repo for changes when it's idle.

**Comment by erikng** [comment]
Right. That's perfectly valid to me. If you'd like, go ahead and close this one.
